packages/xrpl/src/utils/hashes/sha512Half.ts

```diff
@@ -9,11 +10,7 @@ const HASH_SIZE = 64
  * @returns Hash of hex.
  */
 function sha512Half(hex: string): string {
```

Comment:
This might make sense to move to @xrpl/crypto since this is also needed for amendment names.